### PR TITLE
Chplvis

### PR DESCRIPTION
**Pull request by tomyhoi**
Fix a startup file path bug.
Add a settings window with data saved to ~/.cache/chplvis.

### CODE REVIEW

File: tools/chplvis/InfoBar.cxx [hunk 1]
```diff
@@ -34,6 +33,20 @@ static const int Y_OFF = 10;
 static const int Y_OFF = 0;
 #endif
 
+// Local utility functions
+
+// Computation of color for use in displays
+
+Fl_Color heatColor ( double val, double max ) {
```
Comment (ben-albrecht) [inline tools/chplvis/InfoBar.cxx]: small nitpick: A few function definitions / declarations vary stylistically from the rest of the file.

e.g., I would expect:

    Fl_Color heatColor(double val, double max) {

**Comment by ben-albrecht** [comment]
I think a documentation update coupled with this PR would be helpful.

**Comment by ben-albrecht** [comment]
Open question - is there a way we can write tests for GUI features?

**Comment by ben-albrecht** [comment]
Aside from my questions / comment, this change is good to go.

**Comment by tomyhoi** [comment]
I've added documentation changes as suggested by @ben-albrecht .

File: doc/sphinx/source/tools/chplvis/chplvis.rst [hunk 2]
```diff
@@ -520,6 +520,25 @@ If this is set to `false` at compile time
 then :param:`~VisualDebug.VisualDebugOn` must be set to `true`
 on the execution command line to generate :mod:`VisualDebug` data.
 
+Settings
+--------
+
+``chplvis`` has two settings that are set by a settings window and are
+saved in a file named ``~/.cache/chplvis`.   First, the user can
```
Comment (ben-albrecht) [inline doc/sphinx/source/tools/chplvis/chplvis.rst]: Missing a \` here after  `~/.cache/chplvis`

**Pull request merged**